plugin/src/main/avro/schema.avsc

```diff
@@ -12,7 +12,6 @@
     "type" : [ "null", {
       "type" : "record",
       "name" : "QueryCompletedEvent",
-      "namespace" : "io.trino.spi.eventlistener",
```

Comment:
@narape what was the issue with multiple namespaces?

Reply:
I realized that I made the wrong assumption

```scala
import com.spotify.scio._
```
Was the culprit that neither IntelliJ's hotkeys nor myself could import classes from `io.trino.spi.` and any subpackage.  `io` became in scope and scala was complaining that trino was not a valid symbol on `io`

I assumed that the reason of the problem was that the generated classes belonged to the packages already being imported in another jar, like a class loader issue.

I still like it all in the same package but I can revert this last commit